### PR TITLE
2 simple rain drip and liquid splash animations

### PR DESCRIPTION
**Pull request by ItsAnJonth**
Originally these were coloured and looked pretty nice when thrown around but use alpha now so you could have material rain in the future. BLOOD RAIN.
I'm not allowed to write the weather system because I'm bad but I might as well PR these 2 sprites. ;)

Ideally the splashy drip and ripples on water should probably blend both the material and the liquid it's splashing on, as a red splash on blue water would look kinda strange, but eh.

Here's an example of what it looked like when it _did_ have a rudimentary test system behind it and was blue-coloured:

https://cdn.discordapp.com/attachments/314829683780747264/323201569979891712/gooddrips.gif

### CODE REVIEW

File: code/objects/effects/drip.dm [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+	random_state_prefix = ""
+
+	var/amount = 1
+	var/list/splat_images = list()
+
+/obj/effect/random/drip/GetWeight()
+	return max(1,amount)
+
+/obj/effect/random/drip/New()
+	spawn(rand(1,10))
```
Comment (MistakeNot4892) [inline code/objects/effects/drip.dm:14]: Is this missing a line?

File: code/objects/effects/drip.dm [hunk 2]
```diff
@@ -0,0 +1,16 @@
+/obj/effect/random/drip
+	name = "drip"
+	icon = 'icons/objects/effects/rain.dmi'
+	random_states = 1
+	random_state_prefix = ""
+
+	var/amount = 1
+	var/list/splat_images = list()
```
Comment (MistakeNot4892) [inline code/objects/effects/drip.dm:8]: Unremoved from blood decal?

**Comment by ItsAnJonth** [comment]
Did I PR the wrong file...

**Comment by ItsAnJonth** [comment]
I committed the fucking test code not the icon fuck me.

**Comment by MistakeNot4892** [comment]
:+1:

**Pull request closed without merging**